### ISSUE TITLE
Make dock stay underneath all windows, not hide!

### ISSUE THREAD

**Issue by Adam-Antios** [micheleg/dash-to-dock#1272]
I want the dock to stay under all windows. Currently I have configured it to autohide when any windows touch it, but how can I make it static while always being under any new window?

In order to explain, imagine you open the file browser and slowly move it on top of the dock. Then I want the dock to remain static and not autohide, but slowly to become partially hidden by the file manager as it moves on top of it. 

Basically I want the same functionality that all windows have, kind of stacking. If I open two file manager windows, then the one is always going to remain on the bottom of the focused one getting hidden and appearing depending on where it is.

How can I achieve that? It seems weird for this functionality to not being build in, given that this is the default behavior of all windows.
